packages/browser-pool/src/abstract-classes/browser-controller.ts

```diff
@@ -55,6 +55,12 @@ export abstract class BrowserController<
      */
     launchContext: LaunchContext<Library, LibraryOptions, LaunchResult, NewPageOptions, NewPageResult> = undefined!;
 
+    /**
+     * The proxy tier tied to this browser controller.
+     * `undefined` if no tiered proxy is used.
+     */
+    proxyTier : number | undefined = undefined;
```

Comment:
```suggestion
    proxyTier: number | undefined = undefined;
```

also, do we need the initializer? i guess not, so this could be just `proxyTier?: number`

Reply:
Damn, the vacation did me dirty... I remember looking at this forever and thinking this doesn't look quite right 👍🏽